Library/Homebrew/utils/user.rb

```diff
@@ -0,0 +1,12 @@
+require "system_command"
+require "etc"
+
+USER = Etc.getlogin
```

Comment:
Please do not use `Etc.getlogin`. It’s extremely unreliable.

![image](https://user-images.githubusercontent.com/1239874/46293976-0e914300-c595-11e8-83d3-67726658a599.png)

A few seconds later:

![image](https://user-images.githubusercontent.com/1239874/46293998-181aab00-c595-11e8-8d34-033e6871a436.png)